### DISCUSSION TITLE
Custom context size for local models

### DISCUSSION THREAD

**Discussion by gelim** [danny-avila/LibreChat#2586] [Feature Requests & Suggestions]
Hello,

I'm stumbling on the 4096 context size limit on local models served via litellm.

Is there a way/plan to support custom context size limit?

Cheers

**Comment by danny-avila** [comment]
Yes, this week there will be a solution for this.

**Reply by ghost** [reply]
hey @danny-avila how did you fix this?

**Reply by gelim** [reply]
Ping @danny-avila :)

**Reply by danny-avila** [reply]
Sorry forgot to close this discussion.

It's implemented here: 

![image](https://github.com/danny-avila/LibreChat/assets/110412045/00573b54-b2ef-490c-9b38-e297ea2101fe)

You can use presets now for this
